internal/platform/platform.go

```diff
@@ -126,6 +127,10 @@ func init() {
 		name:  "qemu",
 		fetch: qemu.FetchConfig,
 	})
+	configs.Register(Config{
+		name:  "qemu_s390x",
```

Comment:
We don't need/want the arch as part of the platform ID, we can split the qemu fetcher logic according to arch-specific Go buildtags.

Reply:
Right. The change should exist within the QEMU provider and switch on GOARCH - fw_cfg for AMD64 and config-drive for s390x and ppc64.